tests/PublisherTest.php

```diff
@@ -89,43 +57,13 @@ public function testPublishLegacy()
         $this->assertSame('id', $id);
     }
 
-    public function testNetworkIssue()
-    {
-        $httpClient = new MockHttpClient(function (string $method, string $url, array $options = []): ResponseInterface {
-            $this->assertSame('POST', $method);
-            $this->assertSame(self::URL, $url);
-            $this->assertSame(self::AUTH_HEADER, $options['normalized_headers']['authorization'][0]);
-            $this->assertSame('topic=https%3A%2F%2Fdemo.mercure.rocks%2Fdemo%2Fbooks%2F1.jsonld&data=Hi+from+Symfony%21&private=on&id=id&retry=3', $options['body']);
-
-            throw new TransportException('Ops');
-        });
-
-        $provider = new StaticTokenProvider(self::JWT);
-        $hub = new Hub(self::URL, $provider);
-        $publisher = new Publisher($hub, $httpClient);
-
-        $this->expectException(RuntimeException::class);
-        $this->expectExceptionMessage('Failed to send an update.');
-
-        $publisher->publish(new Update(
-            'https://demo.mercure.rocks/demo/books/1.jsonld',
-            'Hi from Symfony!',
-            true,
-            'id',
-            null,
-            3
-        ));
-    }
```

Comment:
Can't we keep this test to prevent regressions in the deprecated area? No blocker, just wondering why removing this one and not the others

Reply:
this one uses the hub, it was added in #35, the test has been moved to `HubTest`